src/custom/constants/mainMenu.ts

```diff
@@ -0,0 +1,77 @@
+import { Routes } from 'constants/routes'
+import { DUNE_DASHBOARD_LINK, CONTRACTS_CODE_LINK, DOCS_LINK, DISCORD_LINK, TWITTER_LINK } from 'constants/index'
+
+// Assets
+import IMAGE_DOCS from 'assets/cow-swap/doc.svg'
+import IMAGE_INFO from 'assets/cow-swap/info.svg'
+import IMAGE_CODE from 'assets/cow-swap/code.svg'
+import IMAGE_DISCORD from 'assets/cow-swap/discord.svg'
+import IMAGE_TWITTER from 'assets/cow-swap/twitter.svg'
+import IMAGE_PIE from 'assets/cow-swap/pie.svg'
+import IMAGE_SLICER from 'assets/cow-swap/ninja-cow.png'
+import IMAGE_GAME from 'assets/cow-swap/game.gif'
+
+export interface MAIN_MENU_TYPE {
```

Comment:
I wonder if these constants should be instead near the menu component, instead of in the general constants

Reply:
In this case the icons are specific to the menu tree structure, which comes from the mainMenu types. So my thinking is it should be closely tied to the menu structure. WDYT?